examples/ErrorHandling/HandlePartialFailures.cs

```diff
@@ -0,0 +1,166 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V1.Common;
+using Google.Ads.GoogleAds.V1.Errors;
+using Google.Ads.GoogleAds.V1.Resources;
+using Google.Ads.GoogleAds.V1.Services;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using static Google.Ads.GoogleAds.V1.Enums.AdGroupCriterionStatusEnum.Types;
+using static Google.Ads.GoogleAds.V1.Enums.KeywordMatchTypeEnum.Types;
+using static Google.Ads.GoogleAds.V1.Errors.ErrorLocation.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V1
+{
+    /// <summary>
+    /// This code example demonstrates how to handle partial failures.
+    /// </summary>
+    public class HandlePartialFailures : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            HandlePartialFailures codeExample = new HandlePartialFailures();
+            Console.WriteLine(codeExample.Description);
+            try
+            {
+                long campaignId = long.Parse("INSERT_CAMPAIGN_ID_HERE");
+
+                // The Google Ads customer ID for which the call is made.
+                long customerId = long.Parse("INSERT_CUSTOMER_ID_HERE");
+
+                codeExample.Run(new GoogleAdsClient(), customerId, campaignId);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("An exception occurred while running this code example. {0}",
+                    ExampleUtilities.FormatException(e));
+            }
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example demonstrates how to handle partial failures.";
+            }
+        }
+
+        /// <summary>
+        /// Runs the code example.
+        /// </summary>
+        /// <param name="client">The Google Ads client.</param>
+        /// <param name="customerId">The customer ID for which the call is made.</param>
+        /// <param name="campaignId">Id of the campaign to which ad groups are added.</param>
+        public void Run(GoogleAdsClient client, long customerId, long campaignId)
+        {
+            // Get the AdGroupServiceClient.
```

Comment:
Nit: in Java we usually write comments so they complete the sentence "This block of code..." not sure if you have the same convention in .net.

So here, it would be "Gets the AdGroupServiceClient".

Below "This ad group should be created successfully" would be "Should be created successfully" etc.

Reply:
What I have here is consistent with rest of the .NET client library.